### DISCUSSION TITLE
Text copy and highlight

### DISCUSSION THREAD

**Discussion by mohabVL** [readium/kotlin-toolkit#374] [Q&A]
How to enable text copy and highlight in radium android

**Comment by mickael-menu** [comment]
Take a look at this EPUB option. By default, it is `true`.

https://github.com/readium/kotlin-toolkit/blob/830812b02a0597b4b22d0d37ec35846db24cb0a4/readium/navigator/src/main/java/org/readium/r2/navigator/epub/EpubNavigatorFragment.kt#L152-L161

**Reply by mohabVL** [reply]
I don't have this disableSelectionWhenProtected option
